src/parquet_copy_hook/copy_to_dest_receiver.rs

```diff
@@ -348,10 +353,10 @@ pub extern "C" fn create_copy_to_parquet_dest_receiver(
     parquet_dest.collected_tuple_count = 0;
     parquet_dest.collected_tuples = std::ptr::null_mut();
     parquet_dest.collected_tuple_column_sizes = std::ptr::null_mut();
-    parquet_dest.row_group_size = row_group_size;
-    parquet_dest.row_group_size_bytes = row_group_size_bytes;
-    parquet_dest.compression = compression;
-    parquet_dest.compression_level = compression_level;
+    parquet_dest.copy_options.row_group_size = row_group_size;
+    parquet_dest.copy_options.row_group_size_bytes = row_group_size_bytes;
+    parquet_dest.copy_options.compression = compression;
+    parquet_dest.copy_options.compression_level = compression_level;
```

Comment:
Is this part of this change, or just an indepedent refactor?

Reply:
sorry for confusion, independent